bitnami/kafka/templates/statefulset.yaml

```diff
@@ -141,7 +212,7 @@ spec:
             - name: KAFKA_ZOOKEEPER_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ if .Values.auth.existingSecret }}{{ .Values.auth.existingSecret }}{{ else }}{{ template "kafka.fullname" . }}{{ end }}
+                  name: {{ include "kafka.secretName" . }}
                   key: kafka-zookeeper-password
```

Comment:
Hi there,

If I set `auth.existingSecret` but not `auth.zookeeperPassword` in `values-producation.yaml`, the `KAFKA_ZOOKEEPER_PASSWORD` won't be enabled. I think the expected behavior is that either (`auth.zookeeperPassword` exist) or (`auth.existingSecret` exist and `kafka-zookeeper-password` is in the secret ) should enable the value `KAFKA_ZOOKEEPER_PASSWORD`